spec/defines/infiniband_interface_spec.rb

```diff
@@ -76,6 +76,60 @@
 
         it { is_expected.to contain_file('/etc/sysconfig/network-scripts/ifcfg-ib0').with_content(my_fixture_read('ifcfg-ib0_with_gateway')) }
       end
+
+      context 'bonding_master => true' do
```

Comment:
This context name doesn't match parameter.

Reply:
true that. I first had bonding_master, but later shortened it for the sake of brevity to bonding only. I'll rename the context.